plasmapy/dispersion/numerical/hollweg_.py

```diff
@@ -0,0 +1,204 @@
+import numpy as np
+import astropy.units as u
+import warnings
+
+from astropy.constants.si import c
+from plasmapy.formulary import parameters as pfp
+from plasmapy.particles import Particle
+from plasmapy.particles.exceptions import ChargeError
+from plasmapy.utils.decorators import validate_quantities
+from plasmapy.utils.exceptions import PhysicsWarning
+
+from typing import Union
```

Comment:
This is one of the things the `pre-commit` check is complaining about.  The import order needs to be done in a specific/consistent way.  There rules for this are defined in the `[tool.isort]` section in the `pyproject.toml` file.  You can always check this yourself locally by using the `isort` package.  So, ...

1. make sure you have `isort==5.7.0` installed (`pip install isort==5.7.0`)
2. once installed, open a terminal and navigate to the root directory of the repo
3. run the following command

    ```bash
    python -m isort --diff --color plasmapy
    ```

   This will run `isort` on the whole `plasmapy` pkg.  If you want to just run it on a specific file then supply that path instead of just `plasmapy`.  The `--diff` option means it will only show you what to change.  If you exclude `--diff` then it'll make the changes for you.  The `--color` option just means it will color the output for (easier to read).